### PR TITLE
feat: add latest page

### PR DESCRIPTION
**Pull request by Aylie-Chou**
This patch add latest page.
To test it, please correspond to [`twreporter-npm-packages` PR #345](https://github.com/twreporter/twreporter-npm-packages/pull/345)

### CODE REVIEW

File: src/containers/Category.js [hunk 1]
```diff
@@ -232,6 +232,30 @@ function postsProp(state, listId, page) {
...
+  const subcategoryList = _.map(CATEGORY_SET[categoryPath], subcategoryKey => {
+    const path = subcategoryKey
+    const label = SUBCATEGORY_LABEL[subcategoryKey]
+    const id = SUBCATEGORY_ID[subcategoryKey]
+    const link =
+      subcategoryKey !== 'all'
+        ? `/categories/${categoryPath}/${path}`
+        : `/categories/${categoryPath}`
+    const isExternal = false
+    return { id, label, link, path, isExternal }
```
Comment (duidae) [inline src/containers/Category.js:250]: `isExternal` is a constant & we only use once, so probably we could just replace this variable with `false` & use comment to show it's a boolean meaning `isExternal`

File: src/containers/Latest.js [hunk 2]
```diff
@@ -0,0 +1,348 @@
...
+    fetchPostsByTagListId,
+    fetchLatestPosts,
+  ])
+
+  // Error handling
+  if (error) {
+    return <SystemError error={error} />
+  }
+
+  const title = 'latest' + siteMeta.name.separator + siteMeta.name.full
```
Comment (duidae) [inline src/containers/Latest.js:120]: 'latest' showed up twice(line 299), so we could replace them with constant variable to reduce duplications

**Review by duidae — CHANGES_REQUESTED** [review]

File: src/containers/Latest.js [hunk 3]
```diff
@@ -0,0 +1,348 @@
...
+
+const TitleTabContainer = styled.div`
+  margin: 64px 0;
+
+  ${mq.tabletOnly`
+    margin: 32px 0;
+  `}
+  ${mq.mobileOnly`
+    margin: 24px 0;
+    width: 86.66666666666667%;
```
Comment (duidae) [inline src/containers/Latest.js:67]: not so important, but curious about this setting, do we really need so many digits? 😂 what about just use 86.67%

File: src/containers/Category.js (hunk 1 above)
Comment (Aylie-Chou) [reply src/containers/Category.js]: updated ([ac2670c](https://github.com/twreporter/twreporter-react/pull/2343/commits/ac2670c5c5e4145142df2d4b7a2c6fbbb804121c))

File: src/containers/Latest.js (hunk 2 above)
Comment (Aylie-Chou) [reply src/containers/Latest.js]: updated ([ac2670c](https://github.com/twreporter/twreporter-react/pull/2343/commits/ac2670c5c5e4145142df2d4b7a2c6fbbb804121c))
`latest` in line 299 means its redux action list id, thus I export new `LATES_LIST_ID` for it (referr to twreporter-npm-packages [#352](https://github.com/twreporter/twreporter-npm-packages/pull/352))

File: src/containers/Latest.js (hunk 3 above)
Comment (Aylie-Chou) [reply src/containers/Latest.js:67]: Good point 😆
It's just the copy code from previous...
I think we could manage it when the layout need to change.

**Review by duidae — APPROVED** [review]

**Pull request merged**